commons/build.gradle

```diff
@@ -172,24 +174,32 @@ task buildAcaPyJavaClient(type: GenerateTask) {
 
 task buildIssuerUiClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-issuer.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-issuer.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/issuer-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
 
 task buildVerifierClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-verifier.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-verifier.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/verifier-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
 
 task buildConnectorUiClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-connector.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-connector.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/connector-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
+
+task setVcmsVersionToContractTemplates(type: Copy) {
```

Comment:
Here I suggest the task configuration API as a Gradle best practice, see:
https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:how_does_it_work 

So this line becomes `tasks.register('setVcmsVersionToContractTemplates', Copy) {.. `

The project is not consistent, we can improve that continously as we work with the tasks.